### PR TITLE
Fast-integration tests with installation via nodejs

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- currently it's being tested by test-infra via pjtester
- based on https://github.com/kyma-project/kyma/pull/10323

### CODE REVIEW

File: tests/fast-integration/installer/charts/ingress-dns-cert/README.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ spec:s
 ```
 
 It also annotates `istio-ingressgateway` with:
+
 ```
-dns.gardener.cloud/class='garden' 
+dns.gardener.cloud/class='garden'
 dns.gardener.cloud/dnsnames='*.$DOMAIN'
 ```
+
```
Comment (pbochynski) [inline tests/fast-integration/installer/charts/ingress-dns-cert/README.md:27]: Something went wrong with rebase/merge?

Comment (aerfio) [reply tests/fast-integration/installer/charts/ingress-dns-cert/README.md:27]: yeah :sweat:

**Review by crabtree — DISMISSED** [review]

**Review by pbochynski — APPROVED** [review]

**Review by pbochynski — DISMISSED** [review]
Installation works stable. No failures related to the installation step: https://github.com/kyma-incubator/local-kyma/actions?query=workflow%3A%22Tests+on+k3s+%28nodejs+installer%29%22

**Review by crabtree — DISMISSED** [review]

**Review by crabtree — APPROVED** [review]

**Pull request merged**